### PR TITLE
fix: updated typo in credit-cards.php credit card image filename

### PR DESCRIPTION
**Pull request by infysumanta**
## What does this PR do?

Its fixed double typo in credit card enum and image avatar

Fixes: https://github.com/appwrite/appwrite/issues/8062

## Test Plan

Tested on local deployment environment

## Related PRs and Issues

- #8062  

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/config/avatars/credit-cards.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
     'amex' => ['name' => 'American Express', 'path' => __DIR__ . '/credit-cards/amex.png'],
     'argencard' => ['name' => 'Argencard', 'path' => __DIR__ . '/credit-cards/argencard.png'],
     'cabal' => ['name' => 'Cabal', 'path' => __DIR__ . '/credit-cards/cabal.png'],
-    'censosud' => ['name' => 'Consosud', 'path' => __DIR__ . '/credit-cards/consosud.png'],
+    'cencosud' => ['name' => 'Consosud', 'path' => __DIR__ . '/credit-cards/cencosud.png'],
```
Comment (stnguyen90) [inline app/config/avatars/credit-cards.php:7]: The name is still incorrect.

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

**Comment by stnguyen90** [comment]
@infysumanta, please also make sure to comment in the original issue to ask to be assigned on the issue.

**Comment by infysumanta** [comment]
Thank you, @stnguyen90. Actually, I am new to the contribution journey. I will make sure to ask before raising a PR and will fix the spelling and update it.

File: app/config/avatars/credit-cards.php (hunk 1 above)
Comment (infysumanta) [reply app/config/avatars/credit-cards.php]: updated the name, thanks

**Review by stnguyen90 — APPROVED** [review]

**Pull request merged**